tests/e2e-openshift/must-gather/check_must_gather.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Create the directory to store must-gather
+MUST_GATHER_DIR=/tmp/must-gather
```

Comment:
Can we use `mktemp -d` here instead of a fixed directory? I know we delete it at the end, but I'd like to minimize potential conflicts.